frontend/apps/filemanager/filemanagerhistory.lua

```diff
@@ -34,6 +38,50 @@ function FileManagerHistory:onSetDimensions(dimen)
     self.dimen = dimen
 end
 
+function FileManagerHistory:printFileInformation(book_props)
+    if book_props.authors == "" then
+        book_props.authors = _("N/A")
+    end
+
+    if book_props.title == "" then
+        book_props.title = _("N/A")
+    end
+
+    if book_props.series == "" then
+        book_props.series = _("N/A")
+    end
+
+    if book_props.pages == "" then
+        book_props.pages = _("N/A")
+    end
+
+    if book_props.language == "" then
+        book_props.language = _("N/A")
+    end
+
+    return {
+        { _("Title") .. " : " .. book_props.title, "" },
+        { _("Authors") .. " : " .. book_props.authors, "" },
+        { _("Series") .. " : " .. book_props.series, "" },
+        { _("Pages") .. " : " .. book_props.pages, "" },
+        { _("Language") .. " : " .. string.upper(book_props.language), "" },
+    }
+end
+
+function FileManagerHistory:fileInformation(file)
+    local file_mode = lfs.attributes(file, "mode")
+    if file_mode == "file" then
```

Comment:
Same as previous comment, use return early pattern can greatly simplify the code here.